endpoint.go

```diff
@@ -360,6 +368,14 @@ func (ep *endpoint) Leave(containerID string, options ...EndpointOption) error {
 		}
 	}
 
+	// Set up non-interface routes.
+	for _, r := range ep.joinInfo.StaticRoutes {
+		err = sb.RemoveStaticRoute(r)
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
Also what about the interface routes? Do they get removed when the interfaces are removed or will that be an error in the kernel when we try to remove the interfaces when there are active routes referencing them?